lasagne/layers/conv.py

```diff
@@ -536,3 +606,176 @@ def convolve(self, input, **kwargs):
         return conved
 
 # TODO: add Conv3DLayer
+
+
+class TransposedConv2DLayer(BaseConvLayer):
+    """
+    lasagne.layers.TransposedConv2DLayer(incoming, num_filters, filter_size,
+    stride=(1, 1), crop=0, untie_biases=False,
+    W=lasagne.init.GlorotUniform(), b=lasagne.init.Constant(0.),
+    nonlinearity=lasagne.nonlinearities.rectify, flip_filters=False, **kwargs)
+
+    2D transposed convolution layer
+
+    Performs the backward pass of a 2D convolution (also called transposed
+    convolution, fractionally-strided convolution or deconvolution in the
+    literature) on its input and optionally adds a bias and applies an
+    elementwise nonlinearity.
+
+    Parameters
+    ----------
+    incoming : a :class:`Layer` instance or a tuple
+        The layer feeding into this layer, or the expected input shape. The
+        output of this layer should be a 4D tensor, with shape
```

Comment:
Yes, I find that confusing as well. We need to change *all* existing docstrings, that should become a separate PR. I'd leave it here for consistency.